src/parser/jobs/drg/modules/Positionals.js

```diff
@@ -0,0 +1,222 @@
+			// did we use the proc? we might not due to invuln windows
+			if (event.ability.guid === ACTIONS.RAIDEN_THRUST.id) {
+				this._rtCombos.used = true
+			}
+
+			// close the window
+			this._rtCombos.push(this._currentCombo)
+			this._currentCombo = null
+		}
+	}
+
+	_onProcGCD(event) {
+		// get action data
+		if (!this._currentCombo) {
+			// if we don't have an open combo window, open one
+			this._currentCombo = {
+				next: NEXT_COMBO[event.ability.guid],
+				success: false,
+				used: false,
+				trueNorthCharges: this._tnCharges,
+				trueNorthCast1: this.combatants.selected.hasStatus(STATUSES.TRUE_NORTH.id),
+				time: event.timestamp,
+			}
+		} else if (this._currentCombo) {
+			// if the action isn't the next expected one, close the current window (not expecting a proc
+			// due to broken combo).
+			if (this._currentCombo.next !== event.ability.guid) {
+				// discard, let the broken combo module yell about this
+				this._currentCombo = null
+			} else {
+				// add the time
+				this._currentCombo.time = event.timestamp
+
+				// check for true north still active
+				this._currentCombo.trueNorthCast2 = this.combatants.selected.hasStatus(STATUSES.TRUE_NORTH.id)
+			}
+		}
+	}
+
+	// called exactly when the timestamp resolves
+	_updateTnCharges() {
+		this._tnCharges += 1
+
+		if (this._tnCharges > TRUE_NORTH_CHARGES) {
+			// clamp to max
+			this._tnCharges = TRUE_NORTH_CHARGES
+		}
+
+		if (this._tnCharges < TRUE_NORTH_CHARGES) {
+			// if we're below the max, queue another charge since the cd will keep ticking
+			this.addTimestampHook(this.parser.currentTimestamp + TRUE_NORTH_CD, this._updateTnCharges)
+		}
+	}
+
+	_tnUsed(event) {
+		// remove charge
+		this._tnCharges -= 1
+
+		// if we're below 0, clamp
+		if (this._tnCharges < 0) {
+			this._tnCharges = 0
+		}
+
+		// mark next recharge time, using the timestamp hook
+		if (!this._nextChargeIn) {
+			this.addTimestampHook(event.timestamp + TRUE_NORTH_CD, this._updateTnCharges)
+		}
+	}
+
+	_procSuccess() {
+		// you did it!
+		if (this._currentCombo) {
+			this._currentCombo.success = true
+		}
+
+		// don't actually close until next gcd happens
+	}
+
+	_onComplete() {
+		const missed = this._rtCombos.filter(combo => !combo.success).length
+
+		this.suggestions.add(
+			new TieredSuggestion({
+				icon: ACTIONS.RAIDEN_THRUST.icon,
+				content: <Trans id="drg.positionals.suggestions.content">Performing the proper positionals for	<ActionLink {...ACTIONS.FANG_AND_CLAW} /> and <ActionLink {...ACTIONS.WHEELING_THRUST} /> will do more damage and	let you use <ActionLink {...ACTIONS.RAIDEN_THRUST} />. Try to land these positionals, and use <ActionLink {...ACTIONS.TRUE_NORTH} /> if	it's available.	</Trans>,
+				tiers: {
+					1: SEVERITY.MINOR,
+					10: SEVERITY.MEDIUM,
+				},
+				value: missed,
+				why: <Trans id="drg.positionals.suggestions.why">You missed {missed} <Plural value={missed} one="proc" other="procs" /> of Raiden Thrust.</Trans>,
```

Comment:
You should move the value into the `<Plural>` tag here with `#` as well:
```js
<Trans id="drg.positionals.suggestions.why">You missed <Plural value={missed} one="# proc" other="# procs" /> of Raiden Thrust.</Trans>
```
This also applies to any other `<Plural>` tags I may have missed. It makes it easier for localization, since the standard order of "[quantity] [noun]" doesn't necessarily apply in other languages.